Makefile

```diff
@@ -0,0 +1,27 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = docs
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+style:
+	isort --profile=black .
+	black .
```

Comment:
What is the opposite of Gemütlichkeit? How about running a set of slow, mutually contradictory linters that throw away CI runs because your files have the wrong number of newlines at the end - focusing on translating working code to equivalent working code - time that could better be spent adding value to the software or doing literally anything else.

Reply:
Also, the makefile runs sphinx.

I enjoy using black and isort because they are fast and automatically fix all problems they find. pre-commit encourages us to add every linter under the sun.